roles/coachview/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+coachview_cds_url: https://10.3.3.207/api/contests/
+coachview_user: coach
+coachview_pass: cds_coachview_password
```

Comment:
Perhaps we should put this in a main.default.yml and ignore the actual file so we don’t have local changes? Not sure what is best

Reply:
we would overwrite these defaults in coachview/vars/main.yml ?